test/src/monitoring_frame_serialization.cpp

```diff
@@ -58,4 +65,26 @@ DynamicSizeRawData serialize(MonitoringFrameMsg& frame)
 
   return raw_processing::toArray<DynamicSizeRawData>(os);
 }
-}  // namespace psen_scan_v2
\ No newline at end of file
+
+std::array<uint8_t, DIAGNOSTIC_DATA_LENGTH_IN_BYTES>
+serializeDiagnosticMessages(std::vector<MonitoringFrameDiagnosticMessage>& messages)
+{
+  std::array<uint8_t, DIAGNOSTIC_DATA_LENGTH_IN_BYTES> raw_diagnostic_data;
+  raw_diagnostic_data.fill(0);
```

Comment:
```
std::array<uint8_t, DIAGNOSTIC_DATA_LENGTH_IN_BYTES> raw_diagnostic_data {};
```
By using value-initialization, all elements get initialized to zero.
So you can delete this line, if you add the curly braces at the end of the line before this one.